jekyll/_cci2/add-ssh-key.md

```diff
@@ -17,30 +17,26 @@ If deploying to your servers requires SSH access, you will need to add SSH keys
 ## Overview
 {: #overview }
 
-There are two reasons to add SSH keys to CircleCI:
+If you are looking to set up an SSH key in order to deploy to GitHub or Bitbucket, refer to the [GitHub](/docs/github-integration/#enable-your-project-to-check-out-additional-private-repositories) or [Bitbucket](/docs/bitbucket-integration/#enable-your-project-to-check-out-additional-private-repositories) integration pages.
```

Comment:
I don't think `deploy to GitHub or Bitbucket` is what we want to say here. Perhaps change the wording to `If you are looking to set up an SSH key in order to check out code from GitHub or Bitbucket...`